lib/internal/webstreams/readablestream.js

```diff
@@ -281,7 +281,9 @@ class ReadableStream {
         this,
         source,
         extractHighWaterMark(highWaterMark, 0));
-      return;
+
+      // eslint-disable-next-line no-constructor-return
+      return makeTransferable(this);
```

Comment:
btw this pattern is used right below this so I copied it from there, rather than separating onto separate lines.

https://github.com/nodejs/node/blob/938341ac61333d898f9aadd17a4fc87373d97f3c/lib/internal/webstreams/readablestream.js#L295-L296